langchain/llms/openai.py

```diff
@@ -95,6 +95,43 @@ def _default_params(self) -> Dict[str, Any]:
         }
         return {**normal_params, **self.model_kwargs}
 
+    def generate(
+        self, prompts: List[str], stop: Optional[List[str]] = None
+    ) -> LLMResult:
+        """Call out to OpenAI's create endpoint.
```

Comment:
"Calls out to OpenAI's endpoint with k unique prompts and returns k * n generations"?